backend/src/v1/code-table/code-table.service.ts

```diff
@@ -431,20 +431,22 @@ export class CodeTableService {
       }
       case "sex": {
         const { data } = await get(token, { 
-          query : "{getAllSexCodes{sex_code short_description long_description display_order active_ind}}"
+          query : "{sexCodes{sexCode shortDescription longDescription displayOrder activeIndicator}}"
         });
-        const results = data.getAllSexCodes.map(
+        const results = data.sexCodes.map(
```

Comment:
there's a better way to do this, and instead of mapping over every type, you should instead use AutoMapper instead

Reply:
I take this back, this is not the time and place to be adding this now as it will require time to handle the missing source types. Best to refactor this after MVP